kuma/settings/common.py

```diff
@@ -57,12 +57,28 @@ def __call__(self, value):
 
 MANAGERS = ADMINS
 
+
+# CONN_MAX_AGE: 'persistent' to keep open connection, or max requests before
+# releasing. Default is 0 for a new connection per request.
+def parse_conn_max_age(value):
+    try:
+        return int(value)
+    except ValueError:
+        assert value.lower() == 'persistent', 'Must be int or "persistent"'
+        return None
+
+
+CONN_MAX_AGE = config('CONN_MAX_AGE', default=60,
```

Comment:
I'm fine with starting at 60 seconds for the default, but I lean towards going even higher (like maybe something between 5 and 15 minutes?), just so we can reduce the overhead of establishing new connections as much as possible.

I suspect that if we go beyond a certain value, we may have to contend with firewalls. In my past experience at two different employers, we had persistent DB connections that connected through a firewall, and discovered that the firewall silently dropped idle connections (wouldn't inform either end) after some period of time (it was something between 30 minutes to an hour if I remember correctly, and of course is configurable per firewall). We got around the issue by pinging the connection periodically to keep it alive.

Reply:
We can play with the settings in prod, and adjust the default as needed. I'd like to only have a minute to shoot ourselves in the foot if, for example, a web pod doesn't properly close a the connection as a new deployment is being rolled out.  We're a little close to the max connections for my comfort...